crates/nu-command/tests/commands/into_filesize.rs

```diff
@@ -68,3 +68,10 @@ fn into_negative_filesize() {
 
     assert!(actual.out.contains("-1 B"));
 }
+
+#[test]
+fn into_positive_filesize() {
+    let actual = nu!("'+1' | into filesize");
+
+    assert!(actual.out.contains("1 B"));
+}
```

Comment:
Because of my previous comment, would you mind adding a few more tests?
- `'+1 KiB' | into filesize`
- `'++1' | into filesize`
- `'++1 KiB' | into filesize`

Thanks!

Reply:
You're right! i missed that point. By your comment, i think we also should re-check minus sign test cases.

Reply:
Feel free to add more test cases if you want.